src/microphone/browser-microphone.ts

```diff
@@ -1,15 +1,12 @@
-import { ErrorCallback } from '../types'
+import { AudioCallback, Microphone, ErrorCallback } from '../types'
```

Comment:
A very small nitpick, but could you please change the file name to `browser_microphone.ts`? Hyphens in file names are rather annoying in terminals.